projects/drivers/jesd204/axi_adxcvr.c

```diff
@@ -80,9 +80,8 @@
 #define ADXCVR_DRP_PORT_ADDR_COMMON		0x00
```

Comment:
We should state the fact that the BROADCAST feature will write the same value to all channels. Since some times not all channels are configured the same (e.g., clock source is different), an update of some register's bits should be made independently for each channel to not overwrite other settings. The BCAST read might also fail since it returns the ORed between all channels.